library/swift/src/filters/ResponseFilter.swift

```diff
@@ -38,6 +38,14 @@ public protocol ResponseFilter: Filter {
   /// - returns: The trailer status containing body with which to continue or buffer.
   func onResponseTrailers(_ trailers: ResponseTrailers) -> FilterTrailersStatus<ResponseTrailers>
 
+  /// Invoked explicitly in response to an asynchronous resume() callback when filter
+  /// iteration has been stopped.
+  /// 
+  /// @return: The resumption status including any previously held entities that remain
+  ///          to be forwarded.
+  /// 
+  func onResumeIteration() -> FilterResumeStatus<ResponseHeaders, ResponseTrailers>
```

Comment:
Are we missing the `FilterResumeStatus` type in this PR?

Reply:
They're coming. :)